packages/rum/test/e2e/async-tests/async.e2e-spec.js

```diff
@@ -50,13 +50,19 @@ describe('async-tests', function () {
      * Check for all types of spans that would be captured by
      * loading script async
      */
-    const spanTypes = ['hard-navigation', 'external']
+    const spanTypes = [
+      'hard-navigation',
+      'external',
+      'longtask',
+      'app',
+      'resource'
+    ]
     /**
-     * Safari does not support Resource & User Timing API
+     * Safari version configured in the test report span type browser-timing
      */
     const { name } = getBrowserInfo()
-    if (name.indexOf('safari') === -1) {
-      spanTypes.push('resource', 'app')
+    if (name.indexOf('safari') !== -1) {
+      spanTypes.push('browser-timing')
```

Comment:
This is interesting, span types should be hard-navigation.browser-timing. Not sure why we are seeing this behavior. 

https://github.com/elastic/apm-agent-rum-js/blob/0862d55a36cc0f906b61ff3289d7fa7416ce4d0a/packages/rum-core/src/performance-monitoring/capture-navigation.js#L98

Reply:
You are right, probably it was a bad interpretation from my side. The test works properly with all the new updates, I removed the safari logic completely 🥳